### DISCUSSION TITLE
Use alpine in tailwind play

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [alpinejs/alpine#2571] [1. Help] [answered]
I tried to use alpine to make a card that can be closed, but when I click the close button nothing happens, here is my code:
https://play.tailwindcss.com/H4GkpcEAUf

**Comment by SimoTod** [comment, marked as answer]
I don't think you can import Alpine in Tailwind play, it just doesn't run at all. You can probably ask them though as it seems to be a Tailwind play limitation.

**Reply by Chandler-Zhu** [reply]
I tried in jsfiddle and it works: https://jsfiddle.net/1mLg9yjh/
